vcd/resource_vcd_network_direct.go

```diff
@@ -39,6 +42,26 @@ func resourceVcdNetworkDirect() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"external_network_gateway": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"external_network_netmask": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"external_network_dns1": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"external_network_dns2": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"external_network_dns_suffix": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
These fields are a nice addition. Why a flat representation was chosen? I mean with `external_network_` prefixes as opposed to all being in a block?

Reply:
For ease of implementation.

Reply:
OK, given that these are **computed** params and would be used in reads only, it would be a burden for the user to to have a block with the current and, most importantly, **single** element (`external_network`).